### DISCUSSION TITLE
Handling DELETE/PUT/PATCH

### DISCUSSION THREAD

**Discussion by MentalGear** [sveltejs/kit#7524] [Q&A]
What is the current (and hopefully future-proof) best practice for JS-free DELETE/PUT/PATCH in SvelteKit ?

I couldn't find an example in the guide, and reviewing the lastest form proposal from Rich (https://github.com/sveltejs/kit/discussions/5875), seems to depreciate `_method`.

**Comment by NormandoHall** [comment]
Maybe you means legacy "endpoints"? Look at [+server.js](https://kit.svelte.dev/docs/routing#server)
